packages/core/src/services/essentials/transitions.ts

```diff
@@ -18,14 +25,16 @@ export class Transitions extends Service {
 	}
 
 	getTransition(
+		selection?: Selection<any, any, any, any>,
```

Comment:
Best to move this parameter at the end (After animate), all 3 params are optional but the order still matters when bundled